prover/src/config.rs

```diff
@@ -94,43 +144,53 @@ impl LayerId {
         }
     }
 
-    pub fn config_path(&self) -> &str {
+    /// The path to the [`Config Parameters`][aggregator::ConfigParams] used to configure the shape
+    /// of the [`Circuit`][halo2_proofs::plonk::Circuit].
+    pub fn config_path(&self) -> PathBuf {
         match self {
-            Self::Layer1 => &LAYER1_CONFIG_PATH,
-            Self::Layer2 => &LAYER2_CONFIG_PATH,
-            Self::Layer3 => &LAYER3_CONFIG_PATH,
-            Self::Layer4 => &LAYER4_CONFIG_PATH,
-            Self::Layer5 => &LAYER5_CONFIG_PATH,
-            Self::Layer6 => &LAYER6_CONFIG_PATH,
+            Self::Layer1 => LAYER1_CONFIG_PATH.to_path_buf(),
+            Self::Layer2 => LAYER2_CONFIG_PATH.to_path_buf(),
+            Self::Layer3 => LAYER3_CONFIG_PATH.to_path_buf(),
+            Self::Layer4 => LAYER4_CONFIG_PATH.to_path_buf(),
+            Self::Layer5 => LAYER5_CONFIG_PATH.to_path_buf(),
+            Self::Layer6 => LAYER6_CONFIG_PATH.to_path_buf(),
             Self::Inner => unreachable!("No config file for super (inner) circuit"),
         }
     }
-}
 
-pub fn asset_file_path(filename: &str) -> String {
-    Path::new(&*ASSETS_DIR)
-        .join(filename)
-        .to_string_lossy()
-        .into_owned()
+    /// Whether or not the [`Snark`][snark_verifier_sdk::Snark] generated at this layer has an
+    /// accumulator.
+    ///
+    /// Every SNARK layer on top of the [`innermost layer`][LayerId::Inner] has an accumulator.
+    pub fn accumulator(&self) -> bool {
```

Comment:
change to `has_accumulator`?

Reply:
It is a planned change. I have not yet removed the `has_accumulator` argument from function calls.